### PR TITLE
fix: pod templates to high

### PR DESCRIPTION
**Pull request by christianhilbrands**
#### Checklist
- [ ] Functionality works & meets specifications (tested on rancher)
- [ ] Templates reviewed
- [ ] Added values to questions
- [ ] Bumped the chart version
- [ ] Updated appVersion (if needed)

### CODE REVIEW

File: charts/molgenis-jenkins/values.yaml [hunk 1]
```diff
@@ -259,19 +259,19 @@ jenkins:
...
               image: "rancher/cli2:v2.4.10"
               command: cat
               args: ""
               ttyEnabled: true
-              resourceRequestCpu: "1"
-              resourceRequestMemory: "1Gi"
-              resourceLimitCpu: "1"
-              resourceLimitMemory: "1Gi"
+              resourceRequestCpu: "100m"
+              resourceRequestMemory: "250Mii"
```
Comment (erikschaberg) [inline charts/molgenis-jenkins/values.yaml:272]: klopt dit de dubbele "ii"

**Review by erikschaberg — COMMENTED** [review]
one typo i think

File: charts/molgenis-jenkins/values.yaml (hunk 1 above)
Comment (christianhilbrands) [reply charts/molgenis-jenkins/values.yaml:272]: goed gespot, ga ik veranderen

Comment (christianhilbrands) [reply charts/molgenis-jenkins/values.yaml]: fixed

**Review by erikschaberg — APPROVED** [review]
LGTM

**Pull request merged**